Dockerfile.amd64

```diff
@@ -1,4 +1,4 @@
-FROM amd64/alpine:3.8
+FROM debian:9.8-slim
 
 LABEL maintainer "Tom Denham <tom@tigera.io>"
```

Comment:
Taking advantage a bit... but while you're in these files, can you update the maintainer label to be `casey@tigera.io`?

Reply:
Done.

Reply:
I changed it for all the places where it had Tom. 

Not sure about the whole `LABEL maintainer "LoZ Open Source Ecosystem (https://www.ibm.com/developerworks/community/groups/community/lozopensource)"`. What is that about?

Reply:
Hey @caseydavenport I think this is the last remaining item. Once we resolve this this is ready for merging.